index.js

```diff
@@ -76,6 +77,19 @@ module.exports = function (str, opts) {
 		urlObj.pathname = urlObj.pathname.replace(/\/{2,}/g, '/');
 	}
 
+	// remove directory index
+	if (opts.removeDirectoryIndex === true) {
+		opts.removeDirectoryIndex = [/^index/];
+	}
+
+	if (Array.isArray(opts.removeDirectoryIndex) && opts.removeDirectoryIndex.length) {
+		var pathComponents = urlObj.pathname.split('/');
+		var lastComponent = pathComponents[pathComponents.length - 1];
+		lastComponent = testParameter(lastComponent, opts.removeDirectoryIndex) ? '' : lastComponent;
+		pathComponents[pathComponents.length - 1] = lastComponent;
```

Comment:
Would be better use as an `if` statement and use `Array#slice` if it's supposed to be removed.